.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This file should get removed (and you should probably add `.vscode/*` to your [global gitignore](https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore)).